component/admin/EventInfo.jsx

```diff
@@ -1,13 +1,14 @@
-import { useRef, useState } from "react";
+import { useRef, useState, useEffect } from "react";
 
 
 export default function EventInfo ({ eventDetails, setEventDetails , eventDetailsError , setEventDetailsError }) {
   const [fileName, setFileName] = useState("");
   const ref = useRef();
   console.log(eventDetails?.eventImage);
-
+  const [pimg, setPimg] = useState()
```

Comment:
Set initial state to empty string rather than nothing or else it would be treated as undefined.